README.md

```diff
@@ -1,2 +1,177 @@
-# module1_project1
-Portfolio: setup and mobile first
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="images/francis_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+
+  <h3><b>Portfolio-setup and Mobile first</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio-setup and Mobile first <a name="about-project"></a>
+
+
+It's a project on building a portfolio and ensuring it is mobile responsive.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+Below are the tech stacks used to develop the website:
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://jquery.com/">JQuery</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+This application is built with the end user in mind. Below are the features implemented:
+
+- **Made for mobile view**
+- **Responsive for desktop view**
+- **Easy to navigate**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+A video to the working of the website is loaded below:
+
+- [Live Demo Link](https://drive.google.com/file/d/1M7w158Yel44sGCPEYjtImRorxx8WnuaO/view?usp=drive_link)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [OPTIONAL] If your project is not hosted and the requirement does not state that it should be hosted, please delete the live demo section or kindly host and include a link to the live version of the project. Your live demo link is currently linked to a project presentation. Please have this fixed. 🤝